lib/template.html

```diff
@@ -65,6 +65,10 @@
   {% assign seo_author_twitter = seo_author_twitter | replace:"@","" %}
 {% endif %}
 
+{% if page.date_modified or page.last_modified_at or page.date %}
+  {% assign seo_date_modified = page.seo.date_modified | default: page.last_modified_at | default: page.date %}
```

Comment:
It's merely a warning. 

That is a valid point that the document's date is not the same as its upload date. 

I will pull that last default snippet to exclude the `page.date`.